application/helpers/LayoutHelper.php

```diff
@@ -445,6 +445,11 @@ protected function beforeSideMenuRender(array $aData)
      */
     public static function renderTopbar(array $aData)
     {
+        App()->getClientScript()->registerScriptFile(
+            App()->getConfig('adminscripts') . 'topbar.js',
```

Comment:
Question : why we don't use assetmanager for adminscripts ?

Reply:
You mean a new package? @Shnoulle

Reply:
No package needed : it's a question and i really thing it's always better to use assetmanager.

`App()->getClientScript()->registerScriptFile(App()->assetManager->publish(App()->getConfig('adminscripts')) . '/' . 'topbar.js');`

For a quick way

Reply:
It's not automatically applied for regsiterScriptFile()?

Reply:
See source in application/core/LSYii_ClientScript.php

Reply:
Arg ! But it's not the best ! It create different asset for all files ! 

Must confirm : but i think it create a copy of all `adminscripts` for each file !

Reply:
OK, well, let me know. :)

Reply:
And report as issue …